go.mod

```diff
@@ -27,22 +25,22 @@ require (
 	github.com/mattn/go-colorable v0.1.7 // indirect
 	github.com/mattn/go-runewidth v0.0.9 // indirect
 	github.com/mattn/go-shellwords v1.0.3
-	github.com/pingcap/br v0.0.0-20200923023944-7456456854e4
+	github.com/pingcap/br v4.0.0-beta.2.0.20201014031603-5676c8fdad1a+incompatible
 	github.com/pingcap/check v0.0.0-20200212061837-5e12011dc712
-	github.com/pingcap/errors v0.11.5-0.20200917111840-a15ef68f753d
+	github.com/pingcap/errors v0.11.5-0.20201029093017-5a7df2af2ac7
 	github.com/pingcap/failpoint v0.0.0-20200702092429-9f69995143ce
-	github.com/pingcap/kvproto v0.0.0-20200916031750-f9473f2c5379
+	github.com/pingcap/kvproto v0.0.0-20201023092649-e6d6090277c9
 	github.com/pingcap/log v0.0.0-20200828042413-fce0951f1463
-	github.com/pingcap/parser v0.0.0-20200924053142-5d7e8ebf605e
-	github.com/pingcap/tidb v1.1.0-beta.0.20200927065602-486e473a86e9
+	github.com/pingcap/parser v0.0.0-20201109022253-d384bee1451e
+	github.com/pingcap/tidb v0.0.0-20201110015039-8d35f17c17f3
```

Comment:
Shall we upgrade tidb/pd to latest master too?

Reply:
Yes, I think so.

Reply:
fine. I will upgrade it later

Reply:
Create a 4.0 PR manually may be better

Reply:
already upgrade tidb/pd to latest master